hello-world/src/lib.rs

```diff
@@ -320,7 +437,8 @@ extern "C" fn init() {
             rested_block,
             allowed_account,
             ft_contract_id,
-            transaction_id
+            transaction_id,
+            reservations,
```

Comment:
No need to set default values explicitly if `Tamagotchi` derives the `Default` trait.
```suggestion
            ..Default::default
```